src/deluge/model/voice/voice.cpp

```diff
@@ -1550,7 +1486,7 @@ skipUnisonPart: {}
 			int32_t* const oscBufferEnd = oscBuffer + (numSamples << 1);
 			// fold
 			if (paramFinalValues[params::LOCAL_FOLD] > 0) {
-				dsp::foldBuffer(oscBuffer, oscBufferEnd, paramFinalValues[params::LOCAL_FOLD]);
+				dsp::foldBufferPolyApproximation(oscBuffer, oscBufferEnd, paramFinalValues[params::LOCAL_FOLD]);
```

Comment:
I don't see this change mentioned in the PR description, what impact does it have?

Reply:
Oops - it uses the non-aliasing fold for stereo rendering. This render path got missed when I added it originally